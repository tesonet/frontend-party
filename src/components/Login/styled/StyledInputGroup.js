import React from 'react';
import styled from 'styled-components';
import { InputGroup } from 'reactstrap';

const StyledContainer = styled(InputGroup)`
  background-color: ${props => props.theme.primaryBgColor};
  margin-bottom: 1.5rem;
`;

const StyledInputGroup = props => <StyledContainer {...props} />;

export default StyledInputGroup;
