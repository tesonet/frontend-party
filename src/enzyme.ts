import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import fetchMock from "jest-fetch-mock";

declare let global: { fetch: {} };

global.fetch = fetchMock;
configure({ adapter: new Adapter() });
