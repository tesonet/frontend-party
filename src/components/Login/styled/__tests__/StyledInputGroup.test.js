import React from 'react';
import { shallow } from 'enzyme';
import { InputGroup } from 'reactstrap';

import StyledInputGroup from '../StyledInputGroup';

describe('StyledInputGroup component', () => {
  it('should render the <InputGroup /> element.', () => {
    const renderedComponent = shallow(<StyledInputGroup />).dive();

    expect(renderedComponent.type()).toEqual(InputGroup);
    expect(renderedComponent).toMatchSnapshot();
  });
});
