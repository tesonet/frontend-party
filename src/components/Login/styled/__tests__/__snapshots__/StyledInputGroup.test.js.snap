// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledInputGroup component should render the <InputGroup /> element. 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Styled(InputGroup) />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "className": "sc-bdVaJa jMSItK",
      "tag": "div",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "className": "sc-bdVaJa jMSItK",
        "tag": "div",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
